sensor/kubernetes/certrefresh/certrepo/service_certificates_repository.go

```diff
@@ -185,7 +185,11 @@ func (r *ServiceCertificatesRepoSecrets) patchServiceCertificate(ctx context.Con
 		Op:    "replace",
 		Path:  "/data",
 		Value: r.secretDataForCertificate(secretSpec, caPem, cert),
-	}}
+	}, {Op: "replace",
+		Path: "/metadata/labels/rhacs.redhat.com/tls",
+		Value: map[string][]byte{
+			"rhacs.redhat.com/tls": []byte("true"),
+		}}}
```

Comment:
1. You need to escape the last `/` with `~1` in `"/metadata/labels/rhacs.redhat.com~1tls"`. See https://www.rfc-editor.org/rfc/rfc6901#section-3 
2. Please test that this will not crash if the secret labels are nil
3. in the `patchSecretDataByteMap` struct, you probably will need to change the `Value` type from `map[string][]byte` to `interface` or `any`